src/error.rs

```diff
@@ -3,13 +3,10 @@ use std::fmt;
 #[derive(Debug)]
 pub enum PngMeError {
     InvalidChunkLength(String),
-    InvalidBytes(String),
```

Comment:
If the digested bytes doesnt correspond to a PNG file header, which error would be used?